docs/src/modules/components/ApiPage/ApiItem.tsx

```diff
@@ -135,7 +135,7 @@ const Root = styled('div')<{ ownerState: { type?: DescriptionType } }>(
       backgroundColor: `var(--muidocs-palette-primary-50, ${lightTheme.palette.primary[50]})`,
     },
     '&>hr': {
-      margin: '20px 0',
+      margin: '24px 0',
```

Comment:
Should it be the smallest the best?

[stripe](https://stripe.com/docs/stripe-js/react#element-components) is at 16px
[shopify](https://polaris.shopify.com/components/selection-and-input/checkbox) at 14px

Reply:
I feel like it gets a bit too cramped. Stripe, for example, in the end, has 32px because each container has 16px ⎯ that's what makes it feel more balanced, I guess. Same with Shopify (28px instead).

<img width="600" alt="Screen Shot 2023-08-02 at 11 07 57" src="https://github.com/mui/material-ui/assets/67129314/c97e2000-f9cc-47d1-b0f9-fee5c0ae174f">
<img width="600" alt="Screen Shot 2023-08-02 at 11 09 59" src="https://github.com/mui/material-ui/assets/67129314/2ab5d313-e987-4f81-a75c-f79a50264cdf">

Reply:
> And the smallest out of the three

From what I understand, there is 24px on each side (top and bottom) so it's the largest of the three.

![image](https://github.com/mui/material-ui/assets/45398769/19dca8e6-d5a1-4303-aa99-a2b8ec1c97fe)

I'm wondering if the need for such space is not due to font-size. Both examples are playing with it to put a hierarchy on information with

- First place the prop name and its type
- Second, its description

Reply:
Ha, you're right 🤦 ended up mixing things up there. 😅  I think though that 18px (36px) looks good, otherwise it still gets too cramped. But in terms of font size, what are you suggesting to change? I'm not sure about reducing the description size to 14px...

Reply:
> But in terms of font size, what are you suggesting to change?

No suggestion, it was just to share some thinking, about why we need more space than others, and because it resonated with Jan feedback. It might make more sens with collapsible interface

> I feel like this would be a lot more legible if:
> - property name comes first, in monospace. followed inline with the description (In Toolpad [file schema](https://mui.com/toolpad/reference/file-schema/) docs we experimented with putting the description above the property, because that is analog to how we tend to do comments in code)
> - basic type/default information goes underneath, collapsed. Expand to show more information (full interface, or function signature,...)
> - there is enough whitespace between the property sections, or horizontal dividers. dividers are used to delineate sections, this is a term + definition list, not a title + paragraph.
> - just monospace + text color is used for code that's in line with text

Reply:
Gotcha, appreciate the clarification! 

I've read his thoughts on Slack and they're indeed great ones! It goes in line with some of the early prototypes we had (both previously and initially on Figma and then the PR you & I collaborated on). Maybe we can pursue these refinements further on your collapsible PR?